oaknational/Oak-Web-Application
# fix: hubspot failure on downloads page [LESQ-563]

## Description

Music year: 2019

- Send hubspot submission before downloads on the download page
- I don't fully understand what the issue was ... 
- I've tested on Chrome, Firefox and Safari (mac) and all seems to be working while on Production I can reproduce the error 100% in Firefox on the downloads page

## Issue(s)

Fixes # 
https://www.notion.so/oaknationalacademy/Hubspot-form-is-sometimes-failing-to-send-647a3bc82c5040db9886deb6ca24f1fa?pvs=4 

## How to test

1. Got to https://www.thenational.academy/teachers/programmes/chemistry-secondary-ks4-l/units/atomic-structure-and-the-periodic-table-u68nqqc/lessons/atoms-elements-and-compounds-h3hasv/downloads?preselected=all on Firefox and download all, see the hubspot failure in the network tab and error in the console
2. Go to https://deploy-preview-2205--oak-web-application.netlify.thenational.academy/teachers/programmes/chemistry-secondary-ks4-l/units/atomic-structure-and-the-periodic-table-u68nqqc/lessons/atoms-elements-and-compounds-h3hasv/downloads?preselected=all and repeat the steps to download
4. You should see hubspot success 

## Screenshots

How it used to look (delete if n/a):
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/6ae9131a-6d54-4a3c-89f7-6833b33ee414" width=500 >


How it should now look:

<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/a59aa906-6465-4ec4-86ca-a75901474b13" width=500 >
